MeretzLib/Apple/iOS/Meretz/Meretz/Meretz+Internal.h

```diff
@@ -0,0 +1,166 @@
+#ifndef __MERETZ_INTERNAL_H__
```

Comment:
In case you're wondering, placing a '+' in the header name is how you ensure that a header file intended to be internal to an XCode static library doesn't get exported with the library build results